src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -323,15 +325,12 @@ public static StringBuilder acquireStringBuilder() {
         if (Jvm.isDebug()) {
             return new StringBuilder();
         } else {
-            // Dangerous, but equivalent to existing behaviour
-            try (final ScopedResource<StringBuilder> scopedResource = SBP.get()) {
-                return scopedResource.get();
-            }
+            return SBP.acquireStringBuilder();
```

Comment:
It's actually not equivalent because the single SB pool is used more broadly than it was. We will retain a specific pool for this use case until the method is deprecated.